src/Actions/PatternsSetup.php

```diff
@@ -222,13 +229,35 @@ public function bcgov_blocks_theme_register_block_patterns() {
 				$keywords           = [];
 
 				if ( ! empty( $categories ) ) {
+
+					// get a list of parent categories.
+					foreach ( $categories as $category ) {
+						if ( 0 === $category->parent ) {
+							$parents[] = $category;
+						}
+					}
+
+					// Register patterns.
 					foreach ( $categories as $category ) {
+						$block_pattern_name = 'bcgov_blocks_theme-' . $category->slug;
+
+						/*
+						 * if the category is a parent, leave the label as-is.
+						 * if the category is a child, prepend its parent's name in parenthesis
+						 */
+						$parent_name = '';
+						foreach ( $parents as $parent ) {
+							if ( $parent->term_id === $category->parent ) {
+								$parent_name = '(' . $parent->name . ')';
+							}
+						}
+
 						if ( ! \WP_Block_Patterns_Registry::get_instance()->is_registered( 'bcgov_blocks_theme-' . $category->slug ) ) {
 							register_block_pattern_category(
-								'bcgov_blocks_theme-' . $category->slug,
+                                'bcgov_blocks_theme-' . $category->slug,
 								[
 									/* translators: %s: category label */
-									'label' => sprintf( __( 'Custom: %s', 'bcgov_blocks_theme' ), $category->name ),
+									'label' => sprintf( __( '%1$s %2$s', 'bcgov_blocks_theme' ), $parent_name, $category->name ),
```

Comment:
This means that all the non parent items will always have a whitespace, better to add the whitespace at end of parent name
```suggestion
									'label' => sprintf( __( '%1$s%2$s', 'bcgov_blocks_theme' ), $parent_name, $category->name ),
```

Reply:
done